.gitlab-ci-prod.yml

```diff
@@ -27,6 +28,32 @@ before_script:
   - scripts/dss-ops.py secrets get gcp-credentials.json > gcp-credentials.json
   - export GOOGLE_APPLICATION_CREDENTIALS=$(pwd -P)/gcp-credentials.json
 
+setup_fusillade:
+  stage: fusillade
+  script:
+    - git clone -b master https://github.com/HumanCellAtlas/dcp-fusillade.git
+    - cd dcp-fusillade
+# currently, there is no environment.prod in the dcp-fusillade repo
+    - source environment # && source environment.prod
+    - cd ..
+    - source environment
+    - if [[ -f "environment.$CI_COMMIT_REF_NAME" ]]; then
+    -   source environment.$CI_COMMIT_REF_NAME
+    - fi
+    - cat ./roles.json | sed "s/\${stage}/${DSS_DEPLOYMENT_STAGE}/g" > temp-roles.json
+    - if [ "$DSS_DEPLOYMENT_STAGE" == "prod" ]; then
+    -   FUS_STAGE='production'
```

Comment:
We should use `prod`.

Reply:
Once fusillade makes the decision to switch to `prod` we can change this file. For now we can just deal, so I don't think this should block.

Reply:
They've made changes.